app/javascript/controllers/datepicker_controller.js

```diff
@@ -20,4 +26,17 @@ export default class extends Controller {
       });
     }
   }
+
+  disconnect() {
+    if (this.datePickerTarget.dataset.datepickerAutosubmit) {
+      this.datePickerTarget.removeEventListener("changeDate", (e) =>
```

Comment:
**suggestion:** Same thing here without the intermediary funciton

Reply:
Same comment as above, doesn't work when updated to this